doc/content/microreactors/s8er/s8er_multiphysics_model.md

```diff
@@ -0,0 +1,245 @@
+| Gap             |   0.0166   |    5193.16    |  0.34   |
+| Cladding        |   8617.93   |   418.68    |  18.85   |
+| Internal Reflector |  1810.08   |    2721.42   |  131.53    |
+
+## Multiphysics Coupling
+
+The multiphysics coupling is done through MOOSE's [MultiApp](https://mooseframework.inl.gov/syntax/MultiApps) system, where Griffin is the main driver and executes Bison as a subapp. In this model the power distribution is solved by Griffin, where it is then transferred to Bison to solve the temperature distribution, where then the temperature distribution is transferred back to Griffin to calculate the new power distribution with updated neutronic data from the temperature feedback. This process is iterated by the `[Executioner]` until the system converges.
+
+!media s8er/multiphysics_flowchart.png
+  caption= Multiphysics Flow Diagram, from [!citep](s8er_naupa2022)
+  style=width:60%;margin-left:auto;margin-right:auto
+
+!listing microreactors/s8er/core_2D_griffin_coupled.i block=MultiApps
+
+The data from each respective model is transfered using MOOSE's [Transfers](https://mooseframework.inl.gov/syntax/Transfers/index.html) system. Since the meshes are tailored for each application, the differences in each mesh must be accounted for in the transfers. As described earlier, the Griffin mesh is generated at the channel level, while the Bison mesh is generated with explicit fuel components modeled. For this reason the [MultiAppProjectionTransfer](https://mooseframework.inl.gov/source/transfers/MultiAppProjectionTransfer.html) is used, which is handy when projecting field variables from one mesh to another. In this case the power generated in the Griffin fuel channel is transfered to only the fuel in Bison. For this reason, a volume normalization factor must be applied to the power density transfered so that power between models is conserved. The normalization factor is simply the ratio of the fuel channel to fuel volume. 
+
+The temperature distribution for each respective tabulation parameter i.e, fuel temperature and coolant temperature is transfered using the [MultiAppInterpolationTransfer](https://mooseframework.inl.gov/source/transfers/MultiAppInterpolationTransfer.html). This transfer is suitable for nodal variables such as variables in the `LAGRANGE` family.   
+
+!listing microreactors/s8er/core_2D_griffin_coupled.i block=Transfers
+
+Reffering back to the `[Materials]` block, this is where the coupled feedback is taken into account. The [CoupledFeedbackMatIDNeutronicsMaterial](https://griffin-docs.hpcondemand.inl.gov/latest/source/materials/CoupledFeedbackMatIDNeutronicsMaterial.html#coupledfeedbackmatidneutronicsmaterial) takes the tabulated neutronic data and uses temperature from bison to interpolate between statepoints. For this reason,  Mutliple transfers for each tabulation parameter are required so that the nuetronic data can be properly interpolated between statepoints.
```

Comment:
```suggestion
Referring back to the `[Materials]` block, this is where the coupled feedback is taken into account. The [CoupledFeedbackMatIDNeutronicsMaterial](https://griffin-docs.hpcondemand.inl.gov/latest/source/materials/CoupledFeedbackMatIDNeutronicsMaterial.html#coupledfeedbackmatidneutronicsmaterial) takes the tabulated neutronic data and uses temperature from bison to interpolate between statepoints. For this reason,  Multiple transfers for each tabulation parameter are required so that the neutronic data can be properly interpolated between statepoints.
```